app/assets/javascripts/modules/calendars/company.es6

```diff
@@ -67,6 +67,13 @@ class CompanyCalendar extends Calendar {
     $.get('/guiders', this.handleResources.bind(this, callback));
   }
 
+  eventClick(event) {
+    if (event.url) {
+      window.open(event.url);
+      return false;
```

Comment:
Copied the example from fullcalendar's docs but preventDefault is fine in this case, so have changed it.